be/src/olap/rowset/segment_v2/segment.cpp

```diff
@@ -73,7 +73,7 @@
 #include "vec/olap/vgeneric_iterators.h"
 
 namespace doris::segment_v2 {
-bvar::Adder<size_t> g_total_segment_num("doris_total_segment_num");
+static bvar::Adder<size_t> g_total_segment_num("doris_total_segment_num");
```

Comment:
BE metrics have a counter `all_segments_num`, is it same as bvar::doris_total_segment_num?